tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx

```diff
@@ -1,18 +1,46 @@
 import React from 'react'
+import axios from 'axios'
 import PropTypes from 'prop-types'
 import { useDispatch, useSelector } from 'react-redux'
 import { fileUploadSections } from '../../reducers/reports'
 import Paginator from '../Paginator'
 import { getAvailableFileList, download } from '../../actions/reports'
 import { useEffect } from 'react'
 import { useState } from 'react'
+import { getParseErrors } from '../../actions/createXLSReport'
 
 const formatDate = (dateStr) => new Date(dateStr).toLocaleString()
 
 const SubmissionHistoryRow = ({ file }) => {
   const dispatch = useDispatch()
 
   const downloadFile = () => dispatch(download(file))
+  const fileName = `${file.year}-${file.quarter}-${file.section}`
+  const returned_errors = () => {
+    getParseErrors(fileParserData.parsedData, fileName)
+  }
+
+  const [fileParserData, setFileParserData] = useState({
+    parsedData: 'parsedData',
+  })
+
+  useEffect(() => {
+    const fileParserErrors = async () => {
+      try {
+        const promise = axios.get(
+          `${process.env.REACT_APP_BACKEND_URL}/parsing/parsing_errors/?file=${file.id}`,
```

Comment:
having this api call inside of `useEffect` means that it will be executed every time the component renders or rerenders - to make it only fire once we could have a `hasLoadedErrorsCsv` state that is checked in `useEffect` before the api call is made. However, given that 4 tables with 5 rows each could show on the page at a time, we might be better off avoiding up to 20 concurrent requests in favor of including a link to the csv file inside the response from `GET` `/api/v1/data_files/`, which is used to populate the entire submission history page. the one request might be a bit slower, but i think it'd be more reliable than the 20 concurrent requests